src/test/java/it/pagopa/interop/probing/eservice/operations/unit/controller/EserviceControllerTest.java

```diff
@@ -394,48 +417,50 @@ void testSearchEservice_whenSizeParameterIsMissing_thenThrows400Exception() thro
   }
 
   @Test
-  @DisplayName("bad request exception is thrown because pageNumber request parameter is missing")
-  void testUpdateEserviceState_whenVersionIdParameterIsMissing_thenThrows400Exception()
-      throws Exception {
-    Mockito.doThrow(BadRequest.class).when(service).searchEservices(Mockito.anyInt(),
-        Mockito.anyInt(), Mockito.anyString(), Mockito.any(), Mockito.anyInt(), Mockito.any());
-    mockMvc
-        .perform(get(apiSearchEserviceUrl).params(getMockRequestParamsUpdateEserviceState("2", null,
-            "Eservice-Name", "Eservice-Version", "false", "ACTIVE")))
-        .andExpect(status().isBadRequest());
+  @DisplayName("the list of ACTIVE e-services has been retrieved")
+  void testGetEservicesActive_whenEservicesIsOnline_thenCheckEservicesIsOnline() throws Exception {
+    Mockito.when(service.getEservicesReadyForPolling(2, 0))
+        .thenReturn(pollingActiveEserviceResponse);
+    MockHttpServletResponse response =
+        mockMvc.perform(get(apiGetEservicesActiveUrl).param("limit", "2").param("offset", "0"))
+            .andReturn().getResponse();
+    assertEquals(response.getStatus(), HttpStatus.OK.value());
+    assertTrue(response.getContentAsString().contains("REST"));
+    assertTrue(response.getContentAsString().contains("1"));
+    assertTrue(response.getContentAsString().contains("base_path_test"));
   }
 
   @Test
-  @DisplayName("e-service last request has been updated")
-  void testUpdateLastRequest_whenGivenValidEservicesRecordId_thenLastRequestIsUpdated()
-      throws Exception {
-    RequestBuilder requestBuilder =
-        MockMvcRequestBuilders.post(String.format(updateLastRequestUrl, eServicesRecordId))
-            .contentType(MediaType.APPLICATION_JSON)
-            .content(mapper.writeValueAsString(changeLastRequest));
-    Mockito.doNothing().when(service).updateLastRequest(updateEserviceLastRequestDto);
-    mockMvc.perform(requestBuilder).andExpect(status().isNoContent());
+  @DisplayName("the list of ACTIVE e-services is empty")
```

Comment:
this needs more context, I suppose something like
```suggestion
  @DisplayName("the retrieve of e-services ready for polling is successful when there are no ready e-services")
```